WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -6,6 +6,11 @@ import enum Yosemite.PointOfSaleItemServiceError
 import struct Yosemite.POSVariableParentProduct
 import class Yosemite.Store
 
+enum POSItemType {
```

Comment:
Based on the discussion p1741916795290219-slack-C070SJRA8DP, lets leave `PointOfSaleItemsController` and service for products, and have a separate one for coupons.

Since Coupons and Products are very different it makes sense to keep the logic separated. The controller doesn't need to know the current selection made in the UI.